src/probnum/filtsmooth/gaussfiltsmooth/gaussfiltsmooth.py

```diff
@@ -75,38 +77,51 @@ def filter(self, dataset, times, **kwargs):
                 **kwargs
             )
             rvs.append(filtrv)
-        return KalmanPosterior(times, rvs, self)
+        return KalmanPosterior(times, rvs, self, with_smoothing=False)
 
-    def smooth(self, filter_posterior, **kwargs):
+    def smooth(self, filter_posterior):
```

Comment:
After seeing `smooth_list` below, what about naming this `smooth_posterior`? If you think it is clear enough that `smooth` operates on a posterior we can leave it as is of course :)

Reply:
To be honest, I am not sure... I like the `smooth()` without `_posterior` because I think it is a nice default. But I would be easily convinced, if you have a strong opinion...

Reply:
I don't have a strong opinion, just wanted to raise the question. Let's keep `smooth` the way it is :)